include/envoy/network/connection.h

```diff
@@ -191,6 +191,11 @@ class Connection : public Event::DeferredDeletable, public FilterManager {
    */
   virtual absl::string_view requestedServerName() const PURE;
 
+  /**
+   * @return requested ECDSA cipher suite(s), if any.
+   */
+  virtual const std::vector<uint16_t>& ecdsaCipherSuites() const PURE;
```

Comment:
I don't think we need this in `Network::Connection`, only in `Network::ConnectionSocket`.